### PR TITLE
added Product Management role on join us page!

### PR DESCRIPTION
**Pull request by ahsu1230**
@martinraison @eishay @stkem

### CODE REVIEW

**Comment by stkem** [comment]
can't say that I like the copy a whole lot, but the code looks good.

File: kifi-backend/modules/shoebox/public/about_us/join_us.html [hunk 1]
```diff
@@ -93,6 +93,24 @@ <h2 class="duration" id="story">Our Story</h2>
...
+<h4>Requirements:</h4>
+
+<ul class="list">
+    <li>Masters or BS in Computer Science or a related field</li>
+    <li>Track record of delivering consumer features across mobile and web</li>
+    <li>3-7 years of experience in Product Management or Product Design</li>
+</ul>
+
+<div class="buttons_row">
+    <a href="mailto:careers@kifi.com?subject=I'm your Product Manager!" target="_blank" class="button blue filter">That's Me</a>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/about_us/join_us.html]: It’d probably be more reliable to encode the spaces in the subject with `+` instead (or `%20`).

**Comment by martinraison** [comment]
@stkem do you mean we should make it a template?  Since we're not really writing or maintaining that code ourselves, I'm not sure if it is better than copy-pasting from Dropbox.

**Comment by 2is10** [comment]
I’m sure he meant “copy” in the “textual content” sense.​

**Comment by stkem** [comment]
@martinraison oh, sorry, I used the term "copy" in this sense: http://en.wikipedia.org/wiki/Copy_(written)
i.e. I think the text is a bit boilerplaty

**Comment by martinraison** [comment]
I see, makes sense. lgtm!

**Comment by ahsu1230** [comment]
merging! (also includes some of Jared's previous comments)

**Pull request merged**